frappe/public/css/docs.css

```diff
@@ -181,7 +181,7 @@
   font-style: normal;
   text-decoration: inherit;
   -webkit-font-smoothing: antialiased;
-  *margin-right: .3em;
+  *margin-right: 0.3em;
```

Comment:
this is invalid css

Reply:
This is auto generated CSS! Remember this is version 10, the build system was diffrent back then. It always causes uncommited changes in frappe app